bounties/BA062304.md

```diff
@@ -4,58 +4,66 @@ title: Rocket Pool Scorecard
+Write or Review the content for a Rocket Pool scorecard in the same theme as the [Lido Scorecard](https://lido.fi/scorecard).
 
-## Requires
-* Must allow the output to be used and built upon. Attribution and share-alike style requirements are ok.
-* Must score Rocket Pool on the same criteria as the Lido scorecard.
-* Must hit both high and low points accurately.
-* Must note the parties affected by any existing risks.
-* Must have acceptable UX.
-* The GMC must approve each milestone.
+## Required Milestones
 
-## Optionals
-* Up to an additional $5,000 available for quality of both UX and content.
+### Milestone B - Content Creation
+**Payout:** 100 - 6,000 USD
 
-## Further Notes
+A large part of this bounty is for Bounty Hunters to determine what information should be included on this scorecard. The goal is to accurately represent Rocket Pool while highlighting its unique strengths in relation to competitors. With that in mind, we provide the following guidelines. 
 
-This bounty must be divided into milestones.
+* Content should be split into separate items with the status 'good', 'okay' or 'needs improvement'.
+* Items should accurately highlight the strengths and weaknesses of the Rocket Pool protocol in its current state.
+* Items should cover common areas of interest and concern for users of the Rocket Pool protocol and its competitors.
+* Items should be written in accessible and easy-to-understand english.
+* Items should cover the level of risk to stakeholder groups within Rocket Pool. This includes:
+  * The protocol DAO (pDAO)
+  * The oracle DAO (oDAO)
+  * The node operators (NOs)
+  * rETH Holders
+  * RPL Holders
+* The amount of compensation will be based on the quality and quantity of content provided.
 
-* **Milestone 1**
-  * A fully constructed mobile-responsive webpage for both the dark and light versions of rocketpool.net
-  * Integration of UX and design requests from a community post on dao.rocketpool.net
-  * Integrated with RP site (not live, but "one-click" away) and easy to maintain or fill in. (I'll contact Nick to see if we set it up to host on GMC github, otherwise we'll use the best workaround.)
-* **Milestone 2**: The creation of content that is unique, high quality, and has been reviewed.
+### Milestone C - Review
+**Payout:** 100 - 2,000 USD
 
-Bounties for milestones are awarded according to the following scheme:
-* **Milestone 1**: $5000
-* **Milestone 2**: $2,000 - $8,000 is allocated to be split among reviewers and content creators. The amount of compensation is based on the quality of the content.
+* Reviews may cover both copy editing and proof-reading. 
+* Reviewers should ensure the factual accuracy of the scorecard.
+* Reviewers should comment on the content selected for inclusion.
+* Reviewers may suggest content for inclusion that they feel is missing.
```

Comment:
I'd probably add a bit more flexibility here:
`Reviewers may offer any suggestions around the content, including: addition of content that they feel is missing, removal of content they feel is low-value, changes to content presentation, etc.